dotcom-rendering/src/components/FeatureCard.tsx

```diff
@@ -377,33 +381,35 @@ export const FeatureCard = ({
 								<div className="image-overlay" />
 
 								<div css={overlayStyles}>
-									<CardHeadline
-										headlineText={headlineText}
-										format={format}
-										fontSizes={headlineSizes}
-										showQuotes={showQuotes}
-										kickerText={
-											format.design ===
-												ArticleDesign.LiveBlog &&
-											!kickerText
-												? 'Live'
-												: kickerText
-										}
-										showPulsingDot={
-											format.design ===
-												ArticleDesign.LiveBlog ||
-											showPulsingDot
-										}
-										byline={byline}
-										showByline={showByline}
-										isExternalLink={isExternalLink}
-										headlineColour={palette(
-											'--feature-card-headline',
-										)}
-										kickerColour={palette(
-											'--feature-card-kicker-text',
-										)}
-									/>
+									<div>
```

Comment:
Can this div wrapper be removed?

Reply:
The extra `div` is so that the headline and byline aren't direct children of the flex container as we don't want space inserted between them. I'll add a comment as it's not obvious why it's there.